### ISSUE TITLE
Is it supporting Amazon Signature V4 for SES?

### ISSUE THREAD

**Issue by elielloyd** [aschroder/Magento-SMTP-Pro-Email-Extension#280]
Hello,
Is this module currently supporting Amazon Signature V4 for SES?
Please let me know.
Thanks.

**Comment by ivancunha** [comment]
Yeah.... Magento 1 stopped sending emails and I think it's because of this! The lack of support for Amazon Signature V4.

Looks like the project was abandoned.... because Magento 1 died... so we have to look for alternatives.....

**Comment by ivancunha** [comment]
If anyone still needs this to work...

Magento-SMTP-Pro-Email-Extension/app/code/local/Aschroder/SMTPPro/lib/AmazonSES.php /
This file was made by someone who also abandoned his project:
https://github.com/christophervalles/Amazon-SES-Zend-Mail-Transport

But someone said they updated the code:
https://github.com/christophervalles/Amazon-SES-Zend-Mail-Transport/issues/7

Here: https://gist.github.com/jezek/d3f9e1b92cbacf990bbcd078c5469fc8

Thanks to jezek :D
